scss/_dropdown.scss

```diff
@@ -9,27 +9,36 @@
 }
 
 .dropdown-toggle {
```

Comment:
IMO, the `.dropdown-toggle` should stay as the Bootstrap one, and IMO we should introduce a `.btn-dropdown` but in another PR ! (maybe #1164 if accepted)